source/libs/utils.js

```diff
@@ -47,6 +47,14 @@ export const domify = html => {
 	return div;
 };
 
+export const getUsername = () => {
+	const username = document.querySelector('.DashUserDropdown-userInfo .username > b');
+
+	if (username) {
```

Comment:
When would the username not be available?

Reply:
if the selector change in the future, I think we prevent some error, or I'm too pessimist? 😊

Reply:
But then we would not know as it would be silently just not showing. We already prevent one feature failing from affecting other, by using the `safely` method.